db/schema.rb

```diff
@@ -167,6 +173,13 @@
     t.index ["user_id"], name: "index_custom_mails_on_user_id"
   end
 
+  create_table "difficulty_levels", force: :cascade do |t|
```

Comment:
Like @tanmoysrt suggested, would be better to rename this table as well to questions_ difficulty_levels 
 https://github.com/CircuitVerse/CircuitVerse/pull/4967/files#r1650731134